tests/Functional/src/com/microsoft/aad/adal/test/DefaultTokenCacheStoreTests.java

```diff
@@ -250,4 +264,116 @@ public void testExpireBuffer() {
     }
+    
+    private void addFRTCacheItem(DefaultTokenCacheStore defaultTokenCacheStore) {
+        final String TEST_AUTHORITY2 = "https://Developer.AndroiD.com/reference/android";
+        UserInfo user2 = new UserInfo("userid2", "givenName", "familyName", "identity", "userid2");
+        TokenCacheItem testFamilyRefreshTokenItemUser2 = new TokenCacheItem();
+        testFamilyRefreshTokenItemUser2.setAccessToken("user2token2Family");
+        testFamilyRefreshTokenItemUser2.setIsMultiResourceRefreshToken(true);
+        testFamilyRefreshTokenItemUser2.setAuthority(TEST_AUTHORITY2);
+        testFamilyRefreshTokenItemUser2.setUserInfo(user2);
+        testFamilyRefreshTokenItemUser2.setFamilyClientId("1");
+        testFamilyRefreshTokenItemUser2.setRefreshToken("user2FRT");
+        testFamilyRefreshTokenItemUser2.setClientId("");
+        testFamilyRefreshTokenItemUser2.setResource("");
+        defaultTokenCacheStore.setItem(CacheKey.createCacheKey(testFamilyRefreshTokenItemUser2),
+                testFamilyRefreshTokenItemUser2);
+    }
+    
+    public void testGetFamilyRefreshTokenItemForUser_withFRTCacheItem() throws IllegalArgumentException, ClassNotFoundException, NoSuchMethodException, InstantiationException, IllegalAccessException, InvocationTargetException{
+        DefaultTokenCacheStore store = (DefaultTokenCacheStore)setupItems();
+        String userId = "userid2";
+        addFRTCacheItem(store);         
+        Method m = ReflectionUtils.getTestMethod(store, "getFamilyRefreshTokenItemForUser", String.class);
+        TokenCacheItem frtTokenCacheItem =(TokenCacheItem)m.invoke(store, userId);
+        assertTrue(frtTokenCacheItem != null);
+    }
+    
+    public void testGetFamilyRefreshTokenItemForUser_noFRTCacheItem() throws IllegalArgumentException, ClassNotFoundException, NoSuchMethodException, InstantiationException, IllegalAccessException, InvocationTargetException{
+        String userId = "userid2";
+        DefaultTokenCacheStore store = (DefaultTokenCacheStore)setupItems();
+        Method m = ReflectionUtils.getTestMethod(store, "getFamilyRefreshTokenItemForUser", String.class);
+        TokenCacheItem frtTokenCacheItem =(TokenCacheItem)m.invoke(store, userId);
+        assertTrue(frtTokenCacheItem == null);
+    }    
+    
+    public void testSerialize_invalidUserId() throws ClassNotFoundException, NoSuchMethodException, InstantiationException, IllegalAccessException, InvocationTargetException{
+        DefaultTokenCacheStore store = (DefaultTokenCacheStore)setupItems();
+        Method m = ReflectionUtils.getTestMethod(store, "serialize", String.class);
+        
+        try {
+            String jsonString = (String)m.invoke(store, "");
+            Assert.fail("not expected");
+        } catch (InvocationTargetException exp){
+             assertTrue(exp.getCause() instanceof IllegalArgumentException);
+        }
+        
+        String testStr = null;
```

Comment:
what is the purpose of having the testStr? why not passing null directly to the following invoke?